src/generator/index.js

```diff
@@ -0,0 +1,20 @@
+const requireGenerator = require("./requireGenerator");
+const parseConfig = require("./parseConfig");
+const writeTemplate = require("./writeTemplate");
+
+/**
+ * Starts generator routine.
+ *
+ * @param {Object} { generatorName, entityName, options } Command object
+ * @param {Object} logger Logger instance
+ */
+module.exports = exports = ({ generatorName, entityName, options }, logger) => {
+  const generator = requireGenerator(generatorName);
+  const generatorConfig = parseConfig(
+    generator.config,
+    Object.assign({}, options, { generator: generator.name, name: entityName })
+  );
+  writeTemplate(generatorConfig);
+  logger.success(`${generator.name} ${entityName} generated successfully`);
```

Comment:
It would be nice to all of the files that were generated along with the path to those files. Especially with custom generators which could potentially place files in un-obvious places.